tests/playwright/src/model/pages/extension-catalog-card-page.ts

```diff
@@ -0,0 +1,62 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+import { expect as playExpect } from '@playwright/test';
+
+import { BasePage } from './base-page';
+import { ExtensionDetailsPage } from './extension-details-page';
+
+export class ExtensionCatalogCardPage extends BasePage {
+  readonly parent: Locator;
+  readonly extensionName: string;
+  readonly detailsButton: Locator;
+  readonly downloadButton: Locator;
+  readonly alreadyInstalledText: Locator;
+
+  constructor(page: Page, extensionName: string) {
+    super(page);
+    this.extensionName = extensionName;
+    this.parent = this.page.getByRole('group', { name: this.extensionName });
+    this.detailsButton = this.parent.getByRole('button', { name: 'More details' });
+    this.downloadButton = this.parent.getByRole('button', { name: 'Install' });
+    this.alreadyInstalledText = this.parent.getByText('Already installed', { exact: true });
+  }
+
+  public async openDetails(): Promise<ExtensionDetailsPage> {
+    await this.parent.scrollIntoViewIfNeeded();
+    await playExpect(this.detailsButton).toBeVisible();
```

Comment:
toBeEnabled() should be used here or whenever following the assert there will be interaction as it is here with the click() command

Reply:
Usually I would agree, except here it is very strange usage of the button where it serves as a link (visually). 
![Screenshot_20240611_081001](https://github.com/containers/podman-desktop/assets/19164299/5eef80e8-4f11-4b9a-8f82-6472d3dd79c9)

Reply:
weird, ok